content/02.intro.md

```diff
@@ -36,30 +37,38 @@ Easy-to-use software packages have brought the techniques of the field out of th
 Additionally, new techniques for fast training have enabled their application to larger datasets [@arxiv:1106.5730].
 Dropout of nodes, edges, and layers makes networks more robust, even when the number of parameters is very large.
 New neural network approaches are also well-suited for addressing distinct challenges.
-For example, neural networks structured as autoencoders or as adversarial networks require no labels and are now regularly used for unsupervised tasks.
```

Comment:
The `New neural network...` sentence feels empty now that this line was moved.  Should we cut it?